Polaris/src/states/State.cpp

```diff
@@ -17,8 +18,8 @@ void State::loop() {
 	loop_impl();
 	this->lastLoopTime = millis();
 	//Sensor stuff here 
-	this->currentState = stateEstimator->onLoop(sensorData); //THis is sus, check pointers and such (tomorrow)
- 
+	//this->currentState = stateEstimator->onLoop(sensorData); //THis is sus, check pointers and such (tomorrow)
+    //Once again- unsure if there is an AttitudeStateEstimator equivalent which should be used Oct 7 2024
```

Comment:
@frostydev99 Is this true as well?

Reply:
The attitude state estimator was constructed in a different way than the other objects that are passed between states IIRC (ideally that should be changed)

That should be a static object only initialized once (that was an issue in the past) and that's why its a pointer. If there is a better way of doing this please let me know!